### ISSUE TITLE
Files without extensions not showing in import window

### ISSUE THREAD

**Issue by sambo57u** [JabRef/jabref#11786]
### JabRef version

Latest development branch build (please note build date below)

### Operating system

Windows

### Details on version and operating system

Fedora L:inux 40 with daily updates

### Checked with the latest development build (copy version output from About dialog)

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. ... Using 5.16 build from 9-17-2024
2. ... When trying to import into current library, the file selection window opens but none of the files without extensions are present (All files is selected). I usually put nee entries into a file called "out".
3. ...

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by Siedlerchr** [comment]
So this must be a OS specific issues then because under mac it shows files without extension
<img width="743" alt="grafik" src="https://github.com/user-attachments/assets/b92201ef-1289-48ed-a475-886b286d83c9">

**Comment by sambo57u** [comment]
Must be. Checked again and it is clearly not showing any file without an extension.

**Comment by Siedlerchr** [comment]
Which desktop environment are you using?

**Comment by sambo57u** [comment]
I am using KDE Plasma 6

**Comment by tina9452** [comment]
hi, I’m new here. Could you please assign this to me? Thank you so much~

**Comment by ThiloteE** [comment]
@tina9452 You can try, but I expect you will have to do a lot of debugging. If you find it too hard, please choose a "free to take" issue from [Good First Issues](https://github.com/orgs/JabRef/projects/5)

**Comment by sambo57u** [comment]
Yes, when importing into current library now all files are shown BUT I noticed that if you just start jabref with no file name and then choose File--->Open library it only shows .bib files, the file selection menu button on the right bottom shows *.bib
with a down arrow to allow for other selections (like show all files) but clicking the arrow does nothing, Thanks!

**Comment by alxafan** [comment]
> Yes, when importing into current library now all files are shown BUT I noticed that if you just start jabref with no file name and then choose File--->Open library it only shows .bib files, the file selection menu button on the right bottom shows *.bib with a down arrow to allow for other selections (like show all files) but clicking the arrow does nothing, Thanks!

Hi @sambo57u,
currently JabRef doesn't allow you to choose different file types in that drop down section, but I don't see a reason for it not to be implemented. However right now there is an issue with invalid files not displaying an error message, which would have to be tackled for this to work cleanly: https://github.com/JabRef/jabref/issues/8625.
For your suggestion I am not sure if you should open a new issue, create a post on the feature section of the [forum](https://discourse.jabref.org/c/features/6) or reopen this issue. My guess is, that it should be opened as a new issue with the enhancement type.
